src/main/telemetry/ibus.c

```diff
@@ -277,12 +282,13 @@ static void dispatchMeasurementRequest(ibusAddress_t address) {
 
     case IBUS_SENSOR_TYPE_TEMPERATURE:
         {
+            //TODO: handle arithmetric better
             #ifdef BARO
-                float temperature = (baroTemperature + 50) / 100.f;
+                uint16_t temperature = (baroTemperature + 50) / 10;
```

Comment:
This logic is just copied from the file: src/main/telemetry/frsky.c
hope that its ok now in the latest commit. I'd love if anyone could try it for real now after all changes.

Thanks for the review by the way!